truss-chains/truss_chains/remote.py

```diff
@@ -351,83 +301,55 @@ def _create_chains_secret_if_missing(remote_provider: b10_remote.BasetenRemote)
         )
 
 
-class _Pusher:
-    class ChainEntry(NamedTuple):
-        service: b10_service.TrussService
-        chainlet_display_name: str
-        is_entrypoint: bool
-
+class _Generator:
```

Comment:
Nit: the name is a bit generic, esp since in python Generator is also a common protocol. Suggestion: `ChainSourceGenerator`.